weblate/utils/render.py

```diff
@@ -36,6 +38,11 @@ def __init__(self, *args, **kwargs):
         super(RestrictedEngine, self).__init__(*args, **kwargs)
 
 
+@register.simple_tag()
```

Comment:
AFAIK this won't work, you need to register the tag in the RestrictedEngine, otherwise it will not work there.

Reply:
I'll do that.

Reply:
In your opinion, where do you think is best place to write the replace tag function?